src/redux/books/books.js

```diff
@@ -0,0 +1,29 @@
+const ADDBOOk = 'book-store/books/ADD BOOK';
+const REMOVEBOOk = 'book-store/books/REMOVE BOOK';
+
+const defaultState = [
+  { genre: 'Comedy', title: 'Harry Potter', author: 'JK Rowling' },
+  { genre: 'Sport', title: 'I am footbAll', author: 'Zlatan Ibrahimovic' },
+];
+
+export default function booksReducer(state = defaultState, action) {
+  switch (action.type) {
+    case ADDBOOk:
+      return [...state, action.book];
+    case REMOVEBOOk:
+      return [
+        ...state.slice(0, action.index),
+        ...state.slice(action.index + 1),
+      ];
```

Comment:
- [Optional] You can also try out the ES6 filter() to create a new array that will not contain the book you want to remove from the store. It is an amazing method.